test/functional/test_ipmi_console.py

```diff
@@ -192,15 +191,15 @@ def setUpClass(cls):
         # FIXME: Need a good way to check if SSH server is listening
         # on port 9300
         cls.ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
-        #Retry SSH to make sure ipmi console started
+        # Retry SSH to make sure ipmi console started
         for i in range(6):
-           try:
-               time.sleep(10)
-               cls.ssh.connect('127.0.0.1', username='', password='', port=9300)
-           except paramiko.ssh_exception.NoValidConnectionsError, e:
-               if i == 5:
-                   raise e
-               continue
+            try:
```

Comment:
Tab in vim:

`:set tabstop=4`, number of visual spaces per TAB
`:set softtabstop=4`, number of spaces in tab when editing
`:set expandtab`, tabs are spaces in further editing
`:retab`, apply current tab setting to existing TAB